modules/web/src/app/cluster/details/external-cluster/external-machine-deployment-details/component.ts

```diff
@@ -169,7 +169,7 @@ export class ExternalMachineDeploymentDetailsComponent implements OnInit, OnDest
   }
 
   private _goBack(): void {
-    this._router.navigateByUrl(`projects/${this.projectID}/clusters/external/${this.cluster.id}`);
+    this._router.navigateByUrl(`projects/${this.projectID}/clusters/externalclusters/${this.cluster.id}`);
```

Comment:
can we use readonly view = View; 

and use `view.ExternalClusters` in **ALL** places